Source/EasyNetQ/EasyNetQ.projitems

```diff
@@ -0,0 +1,184 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
yes I've checked the previous commit and there isn't a `.projitems` at all, so it should be removed.